lms/templates/dashboard/_dashboard_course_listing.html

```diff
@@ -39,113 +39,226 @@
           % if show_courseware_link:
-             % if not is_course_blocked:
-            <a href="${course_target}">${course.display_number_with_default | h} ${course.display_name_with_default}</a>
+            % if not is_course_blocked:
+              <a href="${course_target}">${course.display_name_with_default}</a>
             % else:
-            <a class="disable-look">${course.display_number_with_default | h} ${course.display_name_with_default}</a>
+              <a class="disable-look">${course.display_name_with_default}</a>
             % endif
           % else:
-            <span>${course.display_number_with_default | h} ${course.display_name_with_default}</span>
+            <span>${course.display_name_with_default}</span>
           % endif
         </h3>
-      </hgroup>
+        <div class="course-info">
+          <span class="info-university">${get_course_about_section(course, 'university')} - </span>
+          <span class="info-course-id">${course.display_number_with_default | h}</span>
+          <span class="info-date-block" data-tooltip="Hi">
+          % if course.has_ended():
+            ${_("Ended - {end_date}").format(end_date=course.end_datetime_text("SHORT_DATE"))}
+          % elif course.has_started():
+            ${_("Started - {start_date}").format(start_date=course.start_datetime_text("SHORT_DATE"))}
+          % elif course.start_date_is_still_default: # Course start date TBD
+            ${_("Coming Soon")}
+          % else:   # hasn't started yet
+            ${_("Starts - {start_date}").format(start_date=course.start_datetime_text("SHORT_DATE"))}
+          % endif
+          </span>
+        </div>
+        % if show_courseware_link:
+          <div class="wrapper-course-actions">
+            <div class="course-actions">
+              % if course.has_ended():
+               % if not is_course_blocked:
+                <a href="${course_target}" class="enter-course archived">${_('View Archived Course')}</a>
+              % else:
+                <a class="enter-course-blocked archived">${_('View Archived Course')}</a>
+              % endif
+              % else:
+              % if not is_course_blocked:
+                <a href="${course_target}" class="enter-course">${_('View Course')}</a>
+              % else:
+                <a class="enter-course-blocked">${_('View Course')}</a>
+              % endif
+              % endif
 
+              <!-- <a href="#" data-tooltip="${_('Share on Facebook')}" class="action action-facebook" aria-haspopup="true" aria-expanded="false">
+                <span class="sr">Facebook</span>
+                <i class="fa fa-facebook" aria-hidden="true"></i>
+              </a> -->
+              <!-- <a href="#" data-tooltip="${_('Share on Twitter')}" class="action action-twitter" aria-haspopup="true" aria-expanded="false">
+                <span class="sr">Twitter</span>
+                <i class="fa fa-twitter" aria-hidden="true"></i>
+              </a> -->
```

Comment:
These comments are going to be included in the rendered HTML.  Is this leftover?  If we're planning to enable this down the road, I'd recommend either feature flagging it or removing it and adding it in a future PR.